### ISSUE TITLE
Portal's skybox does not match custom sky in resource pack!

### ISSUE THREAD

**Issue by ghost** [qouteall/ImmersivePortalsModForForge#224]
I have a resource pack enabled which turns the sky pitch black for immersion purposes in a custom map, however this pitch black sky isn't displayed in the skybox of a portal, instead there is a standard light blue skybox.

**Comment by froyo4u** [comment]
this will most likely be marked as low priority

**Comment by benbenlaw** [comment]
yeah 1.16.5 sky effect such as the nether in custom dimensions to not work correctly

**Comment by qouteall** [comment]
What's the datapack that you use?

**Comment by benbenlaw** [comment]
a custom dimension that i made,m if you relog inside the custom dimension the sky effect is ok but when not in that dimension it doesnt work correctly
